fsm_codegen/src/codegen.rs

```diff
@@ -57,7 +58,7 @@ pub fn build_state_store(fsm: &FsmDescription) -> quote::Tokens {
         }
 
         impl #states_store_ty {
-            pub fn new<C>(context: &C) -> #states_store_ty {
+            pub fn new<'a>(context: &#context_ty) -> #states_store_ty {
```

Comment:
`'_` should prevent a warning

Reply:
I did try it already but when using `'_` the compiler throws an error:

```
^^^ `'_` is a reserved lifetime name
```